jdaviz/configs/deconfigged/helper.py

```diff
@@ -0,0 +1,17 @@
+from jdaviz.core.helpers import ConfigHelper
+from jdaviz.configs.default.plugins.line_lists.line_list_mixin import LineListMixin
+
+
+class App(ConfigHelper, LineListMixin):
+    """Base user-facing application helper."""
+
+    _default_configuration = "deconfigged"  # temporary during deconfig process
```

Comment:
Preface: I acknowledge the comment on this line, and I acknowledge that this is the word I started using.

What if we just call this "config"?

Reply:
this should never be seen by the user and will be stored as `self.app.config` which might be confusing if that's set to `'config'` 🤷‍♂️